VEHICLE/NetworkCapEmptyCrowdingVehiclesRemoval.md

```diff
@@ -0,0 +1,33 @@
+---
+ns: VEHICLE
+---
+## NETWORK_CAP_EMPTY_CROWDING_VEHICLES_REMOVAL
+
+```c
+// 0xA4A9A4C40E615885
+void NETWORK_CAP_EMPTY_CROWDING_VEHICLES_REMOVAL(int p0);
```

Comment:
Param is the threshold quantity of vehicles that, when surpassed, will initiate aggressive removal (if activated for this frame).